frontend/widgets/ProductRecommendations/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
I'd say the design is far better than current state of the CSS for `Headline` component. I believe we should adjust it.

Design:
![Screenshot 2019-07-23 at 13 11 34](https://user-images.githubusercontent.com/3393569/61707806-96765280-ad4b-11e9-8d71-9453e1f9632f.png)

Actual Header:
![Screenshot 2019-07-23 at 13 13 07](https://user-images.githubusercontent.com/3393569/61707844-a726c880-ad4b-11e9-927d-6f2c08e935f7.png)

Reply:
changed margins and allow html markup in headline now

Reply:
Not enough. During  the refinement we talked that the “user” should only change background and color, rest should work out of the box.

Even when I added `min-height: 50vw` it doesn't look like on design:
![Screenshot 2019-07-26 at 11 02 45](https://user-images.githubusercontent.com/3393569/61940132-15aa9700-af95-11e9-9a00-5d14f1bee03b.png)

Reply:
And <br> or `\n\ is ok, but I'd say font sizes are not exact